### PR TITLE
pushing code for phases 1 and 2

### PR DESCRIPTION
**Pull request by rjrichards27**
I have pushed the code for phases 1 and 2. The code includes a class for the overall game and code for a sample game in order to test functionality (this would obviously be removed in later versions). Another thing to note is that there is only 82% coverage for the game.py as there is not a way to test the function "prints_dual_pairs" since it only prints and does not return anything (I have asked Patrick about this and he agreed). Please provide comments and possible changes.

### CODE REVIEW

File: data/Harry_Potter_Characters.tsv
Comment (EricR401S) [inline data/Harry_Potter_Characters.tsv:1]: This file is perfect. It contains enough characters, and it includes the creators of this future game.

File: data/Harry_Potter_Spells.tsv
Comment (EricR401S) [inline data/Harry_Potter_Spells.tsv:1]: Potterheads..........You're beyond saving.

File: src/game.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import random
+from dataclasses import dataclass
+
+
+@dataclass
+class Game:
+    """Class for overall game."""
```
Comment (EricR401S) [inline src/game.py:7]: We can expand this docstring at the end of the process. "Class for a tournament style, dueling game between fictional characters from the Harry Potter franchise. " is a suggestion.

File: src/game.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+    def print_duel_pairs(self, character_pairs_list: list[dict[str, str]]) -> None:
+        """Prints duel games before being played."""
+        for duel_pairs in character_pairs_list:
+            print("Duel: ", duel_pairs[0]["Name"], "vs.", duel_pairs[1]["Name"])
+            print("   ")
+
+
+# intitializing character list
+def read_data(file_name: str) -> list[dict[str, str]]:
```
Comment (wafiakmal) [inline src/game.py:28]: this function reminds me of parse_data lol I think it looks good.

File: src/game.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+# reading in data for characters and spells
+hp_spells = "../data/Harry_Potter_Spells.tsv"
+spells_dict = read_data(hp_spells)
+
+# User inputs how many rounds they want played (MAX OF 5 ROUNDS and MIN OF 2)
+rounds_character_num = {5: 32, 4: 16, 3: 8, 2: 4}
+total_rounds = 5
+# generating random list of players for bracket
+selected_chars = random.sample(characters_dict, rounds_character_num[total_rounds])
```
Comment (wafiakmal) [inline src/game.py:54]: we're going to incorporate the random sampling into functions in the next phases, right?

File: src/test_game.py [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+# initializing fake data for tests
+with fake_files(character_fake, spell_fake) as filenames:
+    # testing total length of tuple
+    test_characters = read_data(filenames[0])
+    test_spells = read_data(filenames[1])
+
+print(test_characters)
+
+
+def test_bracket_maker() -> None:
```
Comment (wafiakmal) [inline src/test_game.py:36]: The test is showing 59% for me, but I think it's okay because we're still in phase 1 n 2

**Comment by wafiakmal** [comment]
Personally, I think code looks good for our initial phase!

File: src/game.py (hunk 2 above)
Comment (rjrichards27) [reply src/game.py:28]: Yes, it ultimately does the same thing.

File: src/test_game.py (hunk 4 above)
Comment (rjrichards27) [reply src/test_game.py:36]: Interesting, as when I run it I get 89% for the game.py file.

**Pull request merged**